plugin/adsk/scripts/mayaUsd_preferenceTab.mel

```diff
@@ -139,8 +152,11 @@ global proc mayaUsd_PrefInitOptionVars(int $forceFactoryDefault)
 			-iv mayaUsd_MakePathRelativeToParentLayer false
 			-iv mayaUsd_MakePathRelativeToEditTargetLayer false
 			-iv mayaUsd_MakePathRelativeToImageEditTargetLayer false
+			-iv mayaUsd_ShowDisplayColorTextureOff false
 		;
 	}
+	// Need to refresh the viewport for display mayaUsd_ShowDisplayColorTextureOff
+	ogs -reset;
```

Comment:
I added another viewport refresh here, because if we reset to factoryDefault we need to refresh to show the change